doc/build-windows.md

```diff
@@ -0,0 +1,38 @@
+WINDOWS BUILD NOTES
+====================
+
+Some notes on how to build Bitcoin Core for Windows.
+
+Building on Windows itself is possible, but no one documented the steps to do this.
+
+All developers use cross-compilation to build executables for Windows from Ubuntu. This is also used for the released binaries.
+
+Cross-compilation
+-------------------
+
+These steps can be performed on, for example, an Ubuntu VM. The depends system
+will also work on other Linux distributions, however the commands for
+installing the toolchain will be different.
+
+First install the toolchains:
+
+    sudo apt-get install g++-mingw-w64-i686 mingw-w64-i686-dev g++-mingw-w64-x86-64 mingw-w64-x86-64-dev
+
+To build executables for Windows 32-bit:
+
+    cd depends
+    make HOST=i686-w64-mingw32 -j4
+    cd ..
+    ./configure --prefix=`pwd`/depends/i686-w64-mingw32
+    make
+
+To build executables for Windows 64-bit:
+
+    cd depends
+    make HOST=x86_64-w64-mingw32 -j4
+    cd ..
+    ./configure --prefix=`pwd`/depends/x86_64-w64-mingw32
+    make
+
+For further documentation on the depends system see [../depends/README.md](README.md) in the depends directory.
```

Comment:
Nit: Syntax is wrong. Try `[Read](../link)`

Unrelated note: You may want to remove the `(closes #1401)` from the commit message because GitHub parses this every time this appears in a commit with a different hash (e.g. force push, cherry-pick)